python/ray/worker.py

```diff
@@ -897,7 +906,14 @@ def get_gpu_ids():
     if _mode() == PYTHON_MODE:
         raise Exception("ray.get_gpu_ids() currently does not work in PYTHON "
                         "MODE.")
-    return global_worker.local_scheduler_client.gpu_ids()
+
+    assigned_ids = global_worker.local_scheduler_client.gpu_ids()
+    # If the user had already set CUDA_VISIBLE_DEVICES, then respect that.
```

Comment:
Can you clarify the comment here to specify how `CUDA_VISIBLE_DEVICES` is respected?

Reply:
fixed